README.md

```diff
@@ -148,15 +148,16 @@ Without a Feature Policy, a top-level document and cooperating iframe could recr
 Conversion Registration
 -----------------------
 
-This API will use a similar mechanism for conversion registration as the
+This API will use a similar mechanism for conversion registration similar to the
```

Comment:
nit: maybe drop the first similar in this line

Reply:
Right, fixed.